docs/source/user/elastodyn/input.rst

```diff
@@ -181,7 +181,9 @@ Mass and Inertia
 
 **HubMass**     - Hub mass (kg)
 
-**HubIner**     - Hub inertia about rotor axis [3 blades] or teeter axis [2 blades] (kg m^2)
+**HubIner**     - Hub inertia about rotor axis (2 or 3-blades) (kg m^2)
+
+**HubIner_Teeter**     - Hub inertia about teeter axis (2-blades) (assuming Delta3 angle = 0) (kg m^2)
```

Comment:
I think saying "(assuming Delta3 angle = 0)" is misleading, because HubIner_Teeter is used as the inertia about the teeter axis whether Delta3 is zero or not.  I would also clarify that HubIner_Teeter is only used for 2-bladed rotors.